src/write_http.c

```diff
@@ -807,6 +845,16 @@ static int wh_config_node(oconfig_item_t *ci) /* {{{ */
     wh_callback_free(cb);
     return -1;
   }
+
+  cb->response_buffer_size = WRITE_HTTP_RESPONSE_BUFFER_SIZE;
+  cb->response_buffer = malloc(cb->response_buffer_size);
```

Comment:
Is there a reason to not make response_buffer as array of given size, its length seems to be fixed anyway? It would simplify things in other places too.

Reply:
I think that would be fine.  It's been a while, but I think I was originally trying for being able to grow the response_buffer dynamically if the response coming back wouldn't fit.  I gave up, realizing that for most use cases the first few hundred bytes of the response are all that they need, and even then only in the case of an error.  

I also thought about trying to allocate a response buffer as big as the Content-Length header, but I don't think that value is necessarily available to me here.